pkg/client/proxy_snapshot.go

```diff
@@ -17,21 +16,25 @@ const (
 
 func (c *ProxyClient) VolumeSnapshot(serviceAddress, volumeName string, labels map[string]string) (snapshotName string, err error) {
 	if serviceAddress == "" {
-		return "", errors.Wrapf(ErrParameter, "failed to snapshot volume")
+		return "", errors.Wrap(ErrParameter, "cannot snapshot volume")
```

Comment:
ErrParameter is quite general, could we have a way to improve a bit?

- Make ErrParameter templated
- Or just add more specific info in the message, ex: "failed to snapshot volume due to missing serviceAddress"

Reply:
Addressed.